README.md

```diff
@@ -0,0 +1,17 @@
+# Actix Basics 
+
+Basic `GET` and `POST` example using the endpoints at: `https://jsonplaceholder.typicode.com/<API>`
```

Comment:
Can you add more detail on how to hit these endpoints?  What does the payload look like?  What are the possible response codes?  Also, try providing a curl example.